js/app/controllers/calcontroller.js

```diff
@@ -140,15 +140,18 @@ app.controller('CalController', ['$scope', 'Calendar', 'CalendarService', 'VEven
 				$scope.$apply();
 			});
 		} else {
-			$scope.calendarsPromise = CalendarService.getPublicCalendar(constants.publicSharingToken).then(function(calendar) {
-				$scope.calendars = [calendar];
-				is.loading = false;
-				// TODO - scope.apply should not be necessary here
-				$scope.$apply();
+			constants.publicSharingToken.split(".").forEach(token=>
+			{
+				$scope.calendarsPromise = CalendarService.getPublicCalendar(token).then(function(calendar) {
+					$scope.calendars = [calendar];
+			});
+			is.loading = false;
+			// TODO - scope.apply should not be necessary here
+			$scope.$apply();
 			}).catch((reason) => {
 				angular.element('#header-right').css('display', 'none');
 				angular.element('#emptycontent-container').css('display', 'block');
-			});
+			});		
```

Comment:
Remove spaces added at the end of the line.

Reply:
Is it alright now or do I further look into the indentation ,please tell :)

Reply:
@arunikayadav42 No, its still there. Remove spaces added at the end of the line.